Teams Reinstall - Per User Fix/Teams Per User Install Fix - Remediation.ps1

```diff
@@ -438,7 +438,7 @@ function create-SchedPS1 {
 
         Remove-Item -Path c:\windows\temp\InstallTeams-Remediation.xml -Force -ErrorAction Continue
```

Comment:
Do we need same change for line439 and 440?

Reply:
No, because they typo only exists on line 441.